src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why did this line change?

Reply:
Not entirely sure - possibly CRLF? Content looks the same - do you want me to revert that line anyway?

Reply:
It might be a bight order mark (BOM)

Reply:
Fixed